crates/bvs-vault-router/src/msg.rs

```diff
@@ -25,6 +26,63 @@ pub enum ExecuteMsg {
     /// ExecuteMsg TransferOwnership
     /// See [`bvs_library::ownership::transfer_ownership`] for more information on this field
     TransferOwnership { new_owner: String },
+
+    /// Initiates a slashing request against an active operator of the service (info.sender).
```

Comment:
```suggestion
    /// ExecuteMsg RequestSlashing nitiates a slashing request against an active operator of the service (info.sender).
```

Should start like this, because enum isn't properly formatted for the `schema.go` and .d.ts file.